### ISSUE TITLE
kscrashsentry_reportUserException (Crash)

### ISSUE THREAD

**Issue by SudoPlz** [bugsnag/bugsnag-react-native#174]
Another app crash today:

```javascript
Exception
    /tmp/sandbox/workspace/node_modules/bugsnag-react-native/cocoa/vendor/kscrash/Source/KSCrash/Recording/Sentry/KSCrashSentry_User.c:74:26 kscrashsentry_reportUserException
    /tmp/sandbox/workspace/node_modules/bugsnag-react-native/cocoa/vendor/kscrash/Source/KSCrash/Recording/KSCrash.m:388:5 -[KSCrash reportUserException:reason:language:lineOfCode:stackTrace:terminateProgram:]
    /tmp/sandbox/workspace/node_modules/bugsnag-react-native/cocoa/vendor/bugsnag-cocoa/Source/BugsnagNotifier.m:244:5 -[BugsnagNotifier notify:message:block:]
    /tmp/sandbox/workspace/node_modules/bugsnag-react-native/cocoa/vendor/bugsnag-cocoa/Source/BugsnagNotifier.m:221:5 -[BugsnagNotifier notifyException:block:]
    /tmp/sandbox/workspace/node_modules/bugsnag-react-native/cocoa/vendor/bugsnag-cocoa/Source/Bugsnag.m:83:5 +[Bugsnag notify:block:]
    /tmp/sandbox/workspace/node_modules/bugsnag-react-native/cocoa/BugsnagReactNative.m:144:5 -[BugsnagReactNative notify:]
    Frameworks/CoreFoundation.framework/CoreFoundation ___invoking___
    Frameworks/CoreFoundation.framework/CoreFoundation -[NSInvocation invoke]
    Frameworks/CoreFoundation.framework/CoreFoundation -[NSInvocation invokeWithTarget:]
    /tmp/sandbox/workspace/node_modules/react-native/React/Base/RCTModuleMethod.m:521:3 -[RCTModuleMethod invokeWithBridge:module:arguments:]
    /tmp/sandbox/workspace/node_modules/react-native/React/CxxModule/RCTNativeModule.mm:95:17 invokeInner
    /tmp/sandbox/workspace/node_modules/react-native/React/CxxModule/RCTNativeModule.mm:67:5 operator()
    /usr/lib/system/libdispatch.dylib __dispatch_call_block_and_release
    /usr/lib/system/libdispatch.dylib __dispatch_client_callout
    /usr/lib/system/libdispatch.dylib __dispatch_queue_serial_drain
    /usr/lib/system/libdispatch.dylib __dispatch_queue_invoke
    /usr/lib/system/libdispatch.dylib __dispatch_queue_override_invoke
    /usr/lib/system/libdispatch.dylib __dispatch_root_queue_drain
    /usr/lib/system/libdispatch.dylib __dispatch_worker_thread3
    /usr/lib/system/libsystem_pthread.dylib __pthread_wqthread
    /usr/lib/system/libsystem_pthread.dylib _start_wqthread
```

### Version
`"bugsnag-react-native": "2.3.2",`

Please fix this.

Thank you.

**Comment by fractalwrench** [comment]
Thanks again for another report @SudoPlz - do you know of any reproduction steps for this one? Is this something that occurs regularly?

**Comment by SudoPlz** [comment]
Hi again,

this is [the link of the crash](https://app.bugsnag.com/acuity-scheduling-inc/acuity-scheduling-mobile-app/errors/59e6aaeae5ee3e0019c09f9b?event_id=59e6aaeae5ee3e0019c09f9a&i=tl&m=ci) in bugsnag dashboard.

**Comment by fractalwrench** [comment]
Fixed in v2.5.3
